core/tests/protractor_desktop/explorationImprovementsTab.js

```diff
@@ -123,6 +123,7 @@ describe('Answer Details Improvements', function() {
       creatorDashboardPage.navigateToExplorationEditor();
       explorationEditorPage.navigateToImprovementsTab();
 
+      improvementsTab.verifyTaskCount(1);
```

Comment:
Thanks! Could we also get this for the other e2e describe blocks as well? Also, let's check that the task count drops to 0 after action is taken.

Reply:
Sure! I've added this to other describe blocks as well. 
But I'm unable to push my code due to the following error: 
```
FAILED    Python linting failed
```
```
---------------------------
Checks Not Passed.
---------------------------
Push failed, please correct the linting issues above.
```
I couldn't see any other failure information.

Reply:
Done!